docs/Course Wiki/ECE Course Offerings/ECE120.md

```diff
@@ -0,0 +1,56 @@
+# ECE120
+
```

Comment:
Add a thing here like this:

ECE120 (course title) is a 3-credit-hour course that is specifically required for both EEs and CEs are part of the Electrical and Computer Engineering Core. It is offered in the fall, spring, and summer.

Reply:
Added on latest commit